From 1bfcc99fa69eba7506fa53361cd8714660c1fa1f Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Thu, 14 Apr 2011 14:28:51 +0800
Subject: [PATCH 0086/2463] ENGR00142005-3 ldb: update platform_data
 definition

This patch is used to update platform_data definition to fix build
error and remove unused function.

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 drivers/video/mxc/ldb.c |    8 +-------
 1 files changed, 1 insertions(+), 7 deletions(-)

diff --git a/drivers/video/mxc/ldb.c b/drivers/video/mxc/ldb.c
index 0fbdc5f..f044d04 100644
--- a/drivers/video/mxc/ldb.c
+++ b/drivers/video/mxc/ldb.c
@@ -112,12 +112,6 @@ static bool g_di1_used;
 
 DEFINE_SPINLOCK(ldb_lock);
 
-/*fake clock api*/
-int clk_get_usecount(struct clk *clk)
-{
-	return 1;
-}
-
 struct fb_videomode mxcfb_ldb_modedb[] = {
 	{
 	 "1080P60", 60, 1920, 1080, 7692,
@@ -1244,7 +1238,7 @@ static int ldb_probe(struct platform_device *pdev)
 {
 	int ret = 0;
 	struct resource *res;
-	struct ldb_platform_data *plat_data = pdev->dev.platform_data;
+	struct fsl_mxc_ldb_platform_data *plat_data = pdev->dev.platform_data;
 	uint32_t reg;
 	struct device *temp;
 	int mxc_ldb_major;
-- 
1.7.7.4

